src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -1250,7 +1250,7 @@ public void sendList(CommandSender sender, List<Nation> nations, int page, int t
 		if ((page * 10) > nations.size()) {
 			iMax = nations.size();
 		}

```

Comment:
I know this was out of scope, but can be replaced with `iMax = Math.min(page *10, nations.size());
Same for TownCommand.